axelor-account/src/main/java/com/axelor/apps/account/service/invoice/print/InvoiceProductStatementServiceImpl.java

```diff
@@ -145,7 +146,7 @@ protected String getCorrespondingStatement(
         statement = invoiceProductStatement.getStatement();
       }
     }
-    return statement;
+    return Optional.ofNullable(statement).orElse("");
```

Comment:
```suggestion
    return Optional.ofNullable(statement);
```

I think it is better to return an optional. As a consequence, you can assign the value `null` to the `statement` on [L142](https://github.com/axelor/axelor-open-suite/pull/10203/files#diff-d0e6e9b852693553d1bdba641d84728f2cfd7758db0978f567505d76722d4d36R142).